src/openshift/application.ts

```diff
@@ -24,12 +24,15 @@ export class Application extends OpenShiftItem {
         return null;
     }
 
-    static async getApplicationName() {
+    static async getApplicationName(project: OpenShiftObject) {
+        const applicationList: Array<OpenShiftObject> = await OpenShiftItem.odo.getApplications(project);
         return await vscode.window.showInputBox({
             prompt: "Application name",
```

Comment:
I just realized the prompt message has a different template for each item. We've got 
 - "Application name"
 - "Mention Project name"
 - "Service Name"
 - "Specify the Storage name"

How about we choose one and use it everywhere, for consistency's sake.
 Thing is, I don't particularly like either of these. Personally, I'd be happier to see something in lines of:
 - "Choose a name for the $ITEM"

@mohitsuman @dgolovin feel free to chime in on what you'd like best :)

Reply:
@jrichter1 @mohitsuman I would stick to "Application name" with description "Please provide an Application name" the same pattern is used in `Git: Create Branch...`.